sklearn/datasets/_svmlight_format_io.py

```diff
@@ -135,12 +135,14 @@ def load_svmlight_file(
     Returns
     -------
     X : scipy.sparse matrix of shape (n_samples, n_features)
+        The data matrix.
 
-    y : ndarray of shape (n_samples,), or, in the multilabel a list of
-        tuples of length n_samples.
+    y : ndarray of shape (n_samples,), or a list of tuples of length n_samples
+        The target. The output type depends on the value of the parameter
+        ``multilabel``.
```

Comment:
Can we be explicit and state that a list of tuples is returned when `multilabel=True`?

Reply:
Changed, WDYT ?